src/components/Help/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import style from './style.module.css';
+import help from '../../Images/help.png';
+
+export function Help({ text, color }) {
+    const [displayMessage, setdisplayMessage] = useState('Help ?');
+
+    function handleClick() {
+        setdisplayMessage(text);
+    }
+
+    function colors() {
+        if (color == 'black') {
+            return style.black;
+        } else {
+            return style.white;
+        }
+    }
+
+    return (
+        <div className={style.helpBlock}>
+            <img
+                className={style.helpImage}
+                src={help}
+                onClick={handleClick}
+                text={text}
+            />
+            <div className={colors()}>{displayMessage}</div>
+        </div>
+    );
+}
```

Comment:
Tu peux simplifier un peu:

```suggestion
import React, { useState } from 'react';
import s from './style.module.css';
import help from '../../Images/help.png';

function Help({ text, color }) {
    const [displayMessage, setdisplayMessage] = useState('Help ?');

    return (
        <div className={s.helpBlock}>
            <img
                className={s.helpImage}
                src={help}
                onClick={() => setdisplayMessage(text)}
                text={text}
            />
            <div className={color === 'black' ? s.black : s.white}>{displayMessage}</div>
        </div>
    );
}

export default Help;
```

Reply:
Yes merci  done !